### PR TITLE
Fixed kick URL. Issue #620

### PR DESCRIPTION
**Pull request by antonini**
Fixed kick URL that was reported on issue #620.

### CODE REVIEW

**Comment by karato** [comment]
@antonini I will find a reviewer for your pull requests shortly, thanks for contribution!

**Comment by karato** [comment]
@caarlos0 do this review

**Comment by caarlos0** [comment]
@antonini Thanks! Let's also write a test for that...

**Comment by antonini** [comment]
@caarlos0 Unit test added. Can you review it?

File: netbout-web/src/test/java/com/netbout/rest/TkInboxTest.java [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+ *
+ */
+public final class TkInboxTest {
+
+    /**
+     * TkInbox can kick an User.
+     * @throws Exception If there is some problem inside
+     */
+    @Test
+    public void kickAnUser() throws Exception {
```
Comment (caarlos0) [inline netbout-web/src/test/java/com/netbout/rest/TkInboxTest.java]: @antonini let's rename to `kicksAnUser`

**Comment by antonini** [comment]
@caarlos0 Renamed.

**Comment by caarlos0** [comment]
@rultor merge

**Comment by dmzaytsev** [comment]
@rultor merge

**Comment by karato** [comment]
@elenavolokhova review this ticket please, for compliance with our [quality rules](http://www.teamed.io/qa.html)

**Comment by karato** [comment]
@rultor deploy

**Comment by elenavolokhova** [comment]
@caarlos0 
According to our [quality rules](http://www.teamed.io/qa.html):
 > The code reviewer found at least three problems in the code.
 
 There were only two issues found during review. It didn't improve the code quality much.
Please confirm that you will try to find at least three problems next time or ask to reassign CR to another reviewer.

**Comment by caarlos0** [comment]
@elenavolokhova yes, I'll try..

**Comment by elenavolokhova** [comment]
@karato Quality is acceptable.

**Comment by karato** [comment]
> @karato Quality is acceptable.

@elenavolokhova thank you, let's try to make it "good" next time

**Comment by karato** [comment]
@caarlos0 I added 10 mins to @elenavolokhova (for QA review) in transaction 58966982... Thanks a lot, I just topped your account for **18 mins**, transaction ID `AP-3PH00975LX132740P`, total time was 78 hours and 4 mins... review comments (c=3) added as a bonus... +18 to your rating, your total score is [+2256](http://www.netbout.com/b/35095?open=rating)

**Pull request merged**